instrumentation/rspec/lib/opentelemetry/instrumentation/rspec/formatter.rb

```diff
@@ -0,0 +1,102 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+require 'opentelemetry-api'
+require_relative './version'
+
+module OpenTelemetry
+  module Instrumentation
+    module RSpec
+      # An RSpec Formatter that outputs Otel spans
+      class Formatter
+        attr_reader :output
+
+        ::RSpec::Core::Formatters.register self, :example_started, :example_finished, :example_group_started, :example_group_finished, :start, :stop
+
+        @clock = Time.method(:now)
```

Comment:
wondering what's the difference between `Time.method(:now).call` and `Time.now`?

Reply:
ahh I see, its to save the reference to the original methods if tests alter Time